runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -3266,6 +3266,7 @@ remoteCompile(
    std::string optionsStr = TR::Options::packOptions(compiler->getOptions());
    std::string recompMethodInfoStr = compiler->isRecompilationEnabled() ? std::string((char *) compiler->getRecompilationInfo()->getMethodInfo(), sizeof(TR_PersistentMethodInfo)) : std::string();
 
+   // TODO: make this a synchronized region to avoid bad_alloc exceptions
```

Comment:
Can we put this in an issue that describes what needs to be done? It's not clear what "this" refers to.

Reply:
Opened issue #11349 to address the TODO